README.md

```diff
@@ -1,5 +1,33 @@
 # Køkkenhave app
 
+## Vejledning 22/03-21
+
+* Se udkast til rapport efter påske
```

Comment:
Is this taken from the notes in the notebook or just from memory?

Reply:
From the notebook :)